tests/consensus_tests/test_force_delete_peer.py

```
@@ -0,0 +1,84 @@
+import pathlib
+from .utils import *
```

Ruff (0.8.2)

`from .utils import *` used; unable to detect undefined names

(F403)